compiler/test/grainfmt/patterns.expected.gr

```diff
@@ -18,8 +18,8 @@ match (list) {
 }
 
 match (list) {
-  [_, alongidentalongident123456, _] |
-  [_, _, alongidentalongident123456, _, _] => Some(alongidentalongident123456),
+  [_, alongidentalongident123456, _] | [_, _, alongidentalongident123456, _, _] =>
+    Some(alongidentalongident123456),
```

Comment:
Yeah, this was the type of stuff I was seeing when fixing the 80 character width in my PR